### ISSUE TITLE
install go-micro error

### ISSUE THREAD

**Issue by detectiveHLH** [micro/go-micro#984]
When i install go-micro with command 'go get github.com/micro/go-micro' 

I got erros below.

```shell
go: sigs.k8s.io/yaml@v1.1.0: unrecognized import path "sigs.k8s.io/yaml" (https fetch: Get https://sigs.k8s.io/yaml?go-get=1: dial tcp 35.201.71.162:443: i/o timeout)

go: error loading module requirements
```

Looks like something wrong with the server.

**Comment by milosgajdos** [comment]
Yes, that looks like an intermittent network problem

I'm assuming you have Go modules enabled
```
export GO111MODULE=on
```

And that the version of `go` is at least  `1.13`

**Comment by detectiveHLH** [comment]
The go modules is enabled. But I just found the version of Go is 1.12.

I will try it after I upgrade Go from 1.12 to 1.13

**Comment by detectiveHLH** [comment]
My Bad. I thought my Go version is 1.13.

This problem have been solved after I upgrade go version from 1.12 to 1.13.
